.changeset/wet-keys-occur.md

```diff
@@ -0,0 +1,9 @@
+---
+"@openfun/cunningham-tokens": minor
+"@openfun/cunningham-react": minor
+---
+
+- make scripts cross platform friendly
+- (react) adapt script and config to use vite watch mode
+- fix eslint tsconfig root dir
```

Comment:
Those changes does not rely on "production" codebase so from my point of view, no changelog entries are required for those changes.

Reply:
https://github.com/openfun/cunningham/pull/133/commits/c39dae425ee9a03fb0ab8aae7b6e64c34d62910e#diff-cad685ebfa384252051793fc800876054e645c8ca2c0f1d5aaa134f6d7eb9883L6-L8